tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -97,4 +104,29 @@ def test_nodereplacement_proactive(self, pvc_factory, pod_factory, dc_pod_factor
         self.sanity_helpers.delete_resources()
         # Verify everything running fine
         log.info("Verifying All resources are Running and matches expected result")
-        self.sanity_helpers.health_check()
+        all_pod_obj = pod.get_all_pods(
```

Comment:
self.sanity_helpers.health_check() calls CephCluster.cluster_health_check() which scans the cluster resources and checks their status. 
Why not using it? If something is missing from it, let's add it there

Reply:
for sanity_helpers.health_check() timeout is 60 and first health_check() checks for ceph status and then for pod status. For this test case we need to make sure all pods are in running state and then check ceph status

Reply:
how about modifying sanity_helpers.health_check() so ceph_health_check() will be called after ceph_cluster.cluster_health_check()?

Reply:
This can be done will work on changing health_check()

Reply:
Updated function to check pod status first and then ceph_health

Reply:
> for sanity_helpers.health_check() timeout is 60 and first health_check() checks for ceph status and then for pod status. For this test case we need to make sure all pods are in running state and then check ceph status

Why? If mon or osd stays at pending state after replacement the ceph cluster health will not be healthy.

Reply:
if we do ceph_health_check first we won't get actual result of failure i.e pods reached to running  or not that's why moved health_check down